data/virt_autotest/virt-tests.conf

```diff
@@ -0,0 +1,136 @@
+
+[vmipreset_test]
+GINKGO_FOCUS=vmipreset_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_configuration_test]
+GINKGO_FOCUS=vmi_configuration_test.go
+SOFT_FAIL=test_id:1681
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_headless_test]
+GINKGO_FOCUS=vmi_headless_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[access_test]
+GINKGO_FOCUS=rfe_id:500
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[storage]
+GINKGO_FOCUS=rfe_id:2298
+GINKGO_SKIP=test_id:3108|test_id:3109
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[container_disk_test]
+GINKGO_FOCUS=container_disk_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[datavolume]
+GINKGO_FOCUS=datavolume.go
+GINKGO_SKIP=test_id:3189|test_id:6686|test_id:5252
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[hotplug]
+GINKGO_FOCUS=hotplug.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_cloudinit_test]
+GINKGO_FOCUS=vmi_cloudinit_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_iothreads_test]
+GINKGO_FOCUS=vmi_iothreads_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_multiqueue_test]
+GINKGO_FOCUS=vmi_multiqueue_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[primary_pod_network]
+GINKGO_FOCUS=primary_pod_network.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_multus]
+GINKGO_FOCUS=vmi_multus.go
+EXTRA_OPT=-apply-default-e2e-configuration -skip-shasums-check
+
+[vmi_networking]
+GINKGO_FOCUS=vmi_networking.go
+GINKGO_SKIP=test_id:1780
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[networkpolicy]
+GINKGO_FOCUS=networkpolicy.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_hostdev_test]
+GINKGO_FOCUS=vmi_hostdev_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_gpu_test]
+GINKGO_FOCUS=vmi_gpu_test.go
+GINKGO_SKIP=test_id:4607|test_id:4608
+
+[vmi_monitoring_test]
+GINKGO_FOCUS=vmi_monitoring_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vnc_test]
+GINKGO_FOCUS=vnc_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[console_test]
+GINKGO_FOCUS=console_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[credentials_test]
+GINKGO_FOCUS=credentials_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[expose]
+GINKGO_FOCUS=rfe_id:253
+GINKGO_SKIP=test_id:1538
+EXTRA_OPT=-apply-default-e2e-configuration -skip-dual-stack-test
+
+[replicaset_test]
+GINKGO_FOCUS=replicaset_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[network_services]
+GINKGO_FOCUS=services.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_kernel_boot_test]
+GINKGO_FOCUS=vmi_kernel_boot_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[infra_test]
+GINKGO_FOCUS=infra_test.go
+GINKGO_SKIP=test_id:4099|test_id:4137|test_id:4139|test_id:6535
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[operator_test]
+GINKGO_FOCUS=operator_test.go
+GINKGO_SKIP=test_id:3153
+EXTRA_OPT=-apply-default-e2e-configuration -previous-release-registry=registry.suse.com/suse/sles/15.2 -previous-release-tag=0.40.0-5.14.3 -skip-shasums-check
```

Comment:
The hard code -previous-release-registry and -previous-release-tag won't work for every test requirement. It should either allow external given value during job trigger, or detect the latest/second latest release in registry.suse.com based on test scenario, eg when testing devel repo, we should query the latest, while when testing 15sp4 GA/15SP3 update repo, we should query the second latest.

Reply:
I'd like to introduce two new parameters as below, and it can be passed by openqa test suites. For future, we can consider to detect and get the release version automatically.

PREVIOUS_RELEASE_REGISTRY=registry.suse.com/suse/sles/15.3
PREVIOUS_RELEASE_TAG=0.49.0-150400.1.22